packages/next/src/server-build.ts

```diff
@@ -869,6 +869,7 @@ export async function serverBuild({
           ),
           '___next_launcher.cjs'
         ),
+        operationType: 'SSR', // TODO: Are these only ever SSR?
```

Comment:
This can be inferred from the `group.isPrerenders` and then can add a flag to the API group to signal that as well then otherwise it would be SSR.